org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsi/XSISchemaModel.java

```diff
@@ -167,6 +167,9 @@ public static void computeValueCompletionResponses(ICompletionRequest request,
 		
 		String actualPrefix = document.getSchemaInstancePrefix();
 		DOMAttr attrAtOffset = nodeAtOffset.findAttrAt(offset);
+		if (attrAtOffset == null) {
+			return;
+		}
```

Comment:
The problem with calling `onAttributeValue()` though, is that the request range's offset does not correspond to a `DOMAttr`. Therefore every completion participant that implements `onAttributeValue()` must be careful.

If this is a problem, maybe its a better idea to add new API in `CompletionParticipantAdapter`?  Something along the lines of `onDTDContent()`?

Reply:
Yes it should be a better idea to do that.